examples/bri-3/src/bri/workgroup/workgroups/agents/workgroups.agent.ts

```diff
@@ -21,6 +21,7 @@ export class WorkgroupAgent {
     participants: BpiSubject[],
     worksteps: Workstep[],
     workflows: Workflow[],
+    status: WorkgroupStatus,
```

Comment:
As mentioned, new workgroup will always be in Active status initially, no need to pass it here.